tests/http/simple-spin-rust/spin.toml

```diff
@@ -5,14 +5,11 @@ name = "spin-hello-world"
 trigger = {type = "http", base = "/test"}
 version = "1.0.0"
 
-[config]
-object = { default = "teapot" }
-
 [[component]]
 id = "hello"
 source = "target/wasm32-wasi/release/spinhelloworld.wasm"
 files = [ { source = "assets", destination = "/" } ]
 [component.trigger]
 route = "/hello/..."
 [component.config]
-message = "I'm a {{object}}"
+message = "I'm a teapot"
```

Comment:
This config still works for `spin up`, but not for `spin deploy`. The issue toml `ToModel` is in hippo.

```
fail: Hippo.Application.Revisions.Commands.ImportRevisionsCommand[0]
      Hippo Request: Unhandled Exception for Request ImportRevisionsCommand Hippo.Application.Revisions.Commands.ImportRevisionsCommand
      Tomlyn.TomlException: (4,2) : error : Unable to set the property config on object type Hippo.Application.Common.Interfaces.BindleService.RevisionSpinToml.

         at Tomlyn.Toml.ToModel[T](DocumentSyntax syntax, TomlModelOptions options)
         at Hippo.Infrastructure.Services.BindleService.ParseSpinTomlParcel(HttpContent parcelResponse) in /home/runner/work/hippo/hippo/src/Infrastructure/Services/BindleService.cs:line 62
         at Hippo.Infrastructure.Services.BindleService.GetSpinTomlParcel(Invoice invoice, String revisionId) in /home/runner/work/hippo/hippo/src/Infrastructure/Services/BindleService.cs:line 50
         at Hippo.Infrastructure.Services.BindleService.GetRevisionDetails(String revisionId) in /home/runner/work/hippo/hippo/src/Infrastructure/Services/BindleService.cs:line 27
         at Hippo.Application.Revisions.EventHandlers.UpdateRevisionDetails.Handle(CreatedEvent`1 notification, CancellationToken cancellationToken) in /home/runner/work/hippo/hippo/src/Application/Revisions/EventHandlers/UpdateRevisionDetails.cs:line 33
         at MediatR.Mediator.PublishCore(IEnumerable`1 allHandlers, INotification notification, CancellationToken cancellationToken)
         at MediatR.MediatorExtensions.DispatchDomainEvents(IMediator mediator, DbContext context) in /home/runner/work/hippo/hippo/src/Infrastructure/Common/MediatorExtensions.cs:line 22
         at Hippo.Infrastructure.Data.ApplicationDbContext.SaveChangesAsync(CancellationToken cancellationToken) in /home/runner/work/hippo/hippo/src/Infrastructure/Data/ApplicationDbContext.cs:line 60
         at Hippo.Application.Revisions.Commands.ImportRevisionsCommandHandler.Handle(ImportRevisionsCommand request, CancellationToken cancellationToken) in /home/runner/work/hippo/hippo/src/Application/Revisions/Commands/ImportRevisionsCommand.cs:line 49
         at Hippo.Application.Common.Behaviours.PerformanceBehaviour`2.Handle(TRequest request, CancellationToken cancellationToken, RequestHandlerDelegate`1 next) in /home/runner/work/hippo/hippo/src/Application/Common/Behaviours/PerformanceBehaviour.cs:line 31
         at Hippo.Application.Common.Behaviours.ValidationBehaviour`2.Handle(TRequest request, CancellationToken cancellationToken, RequestHandlerDelegate`1 next) in /home/runner/work/hippo/hippo/src/Application/Common/Behaviours/ValidationBehaviour.cs:line 35
         at Hippo.Application.Common.Behaviours.AuthorizationBehaviour`2.Handle(TRequest request, CancellationToken cancellationToken, RequestHandlerDelegate`1 next) in /home/runner/work/hippo/hippo/src/Application/Common/Behaviours/AuthorizationBehaviour.cs:line 78
         at Hippo.Application.Common.Behaviours.UnhandledExceptionBehaviour`2.Handle(TRequest request, CancellationToken cancellationToken, RequestHandlerDelegate`1 next) in /home/runner/work/hippo/hippo/src/Application/Common/Behaviours/UnhandledExceptionBehaviour.cs:line 19
```

Reply:
I file an issue for it.

https://github.com/deislabs/hippo/issues/978

Reply:
I wonder if we should leave this test project as-is (so that it is continued to be tested via the other integration tests) and create a separate test project that is compatible with hippo/`spin deploy` today?  We could have a link to the issue in deislabs/hippo in the new test project for context.  WDYT?

Reply:
👍 Yup, that sounds good to me.